package/kedro_viz/models/utils.py

```diff
@@ -39,3 +45,45 @@ def get_dataset_type(dataset: "AbstractDataset") -> str:
     abbreviated_module_name = ".".join(dataset.__class__.__module__.split(".")[-2:])
     class_name = f"{dataset.__class__.__qualname__}"
     return f"{abbreviated_module_name}.{class_name}"
+
+
+def get_file_size(file_path: Union[str, None]) -> int:
+    """Get the dataset file size using fsspec. If the file_path is a directory,
+    get the latest file created (this corresponds to the latest run)
+
+    Args:
+        file_path: The file path for the dataset
+    """
+    try:
+        if not file_path:
+            return 0
```

Comment:
Should it be None instead? How would front end handle this?

Reply:
In the frontend, if the file size is returned as 0 bytes, we will be showing N/A

Reply:
Could it be return `None` here? then in the frontend use if not filesize since 0 or None would both be treated as `False`